src/util/hooks/useBodyRef.ts

```diff
@@ -0,0 +1,12 @@
+'use client';
+import { useEffect, useState } from 'react';
+
+export const useBodyRef = () => {
+	const [bodyRef, setBodyRef] = useState<HTMLElement | null>(null);
+
+	useEffect(() => {
+		setBodyRef(document.body);
+	}, []);
+
+	return bodyRef;
+};
```

Comment:
tohle mi smrdí... k čemu to je? nějak jsem si nevšiml že bys to někde použil

Reply:
som zabudol